runtime/JavaScript/src/antlr4/BufferedTokenStream.js

```diff
@@ -83,6 +83,10 @@ export default class BufferedTokenStream extends TokenStream {
 		this.index = this.adjustSeekIndex(index);
 	}
 
+	get size() {
+		return this.tokens.length;
+	}
```

Comment:
This attribute is used by the `TokenStreamRewriter`. Seems that it was missing so I added it. In the Java run-time, this attribute is defined on the `IntStream` interface and subsequently implemented in `BufferedTokenStream.java`.